src/app/shared/components/template/template-container.component.ts

```diff
@@ -468,7 +462,7 @@ export class TemplateContainerComponent implements OnInit, OnDestroy, ITemplateC
   private subscribeToQueryParamChanges() {
     this.route.queryParams
       .pipe(takeUntil(this.componentDestroyed$))
-      .subscribe(async (params: IQueryParams) => {
+      .subscribe(async (params: any) => {
```

Comment:
I like the way the debug toggle just changes the query parameter so we can trigger it from the URL as well